pom.xml

```diff
@@ -161,6 +161,25 @@
                     <artifactId>maven-clean-plugin</artifactId>
                     <version>${maven-clean-plugin.version}</version>
                 </plugin>
+          <plugins>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-jar-plugin</artifactId>
+            <version>3.2.2</version>
+            <configuration>
+                <archive>
+                    <manifest>
+                        <addClasspath>true</addClasspath>
+                        <mainClass>your.main.class</mainClass>
+                    </manifest>
+                    <manifestEntries>
+                        <!-- Add other manifest entries if needed -->
+                    </manifestEntries>
+                </archive>
+                <signaturesFiles>
+                    <signaturesFile><your-signatures-file></signaturesFile>
+                </signaturesFiles>
+            </configuration>
+        </plugin>
```

Comment:
This is the wrong plugin to add.  Go to the `forbiddenapis` plugin here:
https://github.com/oshi/oshi/blob/2a37177fe9c9c5c96f6711b62d7b8c97814f0d09/pom.xml#L746-L770

You'll be adding the `<signatureFiles>` section inside configuration.  See sample POM entry [here](https://github.com/policeman-tools/forbidden-apis/wiki/MavenUsage#maven-usage-instructions).  Format of that file (you can put it in the `config` directory under root) is [here](https://github.com/policeman-tools/forbidden-apis/wiki/SignaturesSyntax#syntax-of-forbidden-signatures).